contracts/transfer/src/wasm/internal.rs

```diff
@@ -177,28 +177,35 @@ impl<S: Store> TransferContract<S> {
 
     pub(crate) fn take_crossover(
         &mut self,
-    ) -> Result<(Crossover, PublicKey), S::Error> {
+    ) -> Result<(Crossover, PublicKey), Error> {
         let crossover =
-            self.var_crossover.take().ok_or(InvalidEncoding.into())?;
-        let pk = self.var_crossover_pk.take().ok_or(InvalidEncoding.into())?;
+            self.var_crossover.take().ok_or(Error::CrossoverNotFound)?;
+
+        let pk = self
+            .var_crossover_pk
+            .take()
+            .ok_or(Error::CrossoverNotFound)?;
 
         Ok((crossover, pk))
     }
 
-    pub(crate) fn assert_payable(_address: &BlsScalar) -> Result<(), S::Error> {
+    pub(crate) fn assert_payable(_address: &BlsScalar) -> Result<(), Error> {
         //  FIXME Use isPayable definition
         //  https://github.com/dusk-network/rusk-vm/issues/151
 
         Ok(())
     }
 
     pub(crate) fn assert_proof(
-        proof: Vec<u8>,
-        vd: &[u8],
-        pi: Vec<PublicInput>,
-    ) -> Result<(), S::Error> {
+        _proof: Vec<u8>,
+        _vd: &[u8],
+        _pi: Vec<PublicInput>,
+    ) -> Result<(), Error> {
+        /*
         rusk_abi::verify_proof(proof, vd.to_vec(), pi)
             .then(|| ())
-            .ok_or(InvalidEncoding.into())
+            .ok_or(Error::ProofVerificationError)
+            */
+        Ok(())
     }
 }
```

Comment:
Is really needed?

Reply:
And if the interface is, why is not being called?

Reply:
The proof verification method constantly changes, so that is a way to encapsulate all these changes into a single function

Reply:
But shouldn't be commented out, that is a mistake. Will enable again

Reply:
Fixed